tools/build/pack_cordova_sample.py

```diff
@@ -427,6 +427,82 @@ def packSampleApp(app_name=None):
     os.chdir(orig_dir)
     return True
 
+def packMobileSpec_cli(app_name=None):
+    project_root = os.path.join(BUILD_ROOT, app_name)
+    output = commands.getoutput("cordova -v")
+    if output != "5.0.0":
+        LOG.error("Cordova 4.0 build requires Cordova-CLI 5.0.0, install with command: '$ sudo npm install cordova@5.0.0 -g'")
+        return False
+
+    plugin_tool = os.path.join(BUILD_ROOT, "cordova-plugin-crosswalk-webview")
+    if not doCopy(os.path.join(BUILD_PARAMETERS.pkgpacktools, "cordova_plugins", "cordova-plugin-crosswalk-webview"), plugin_tool):
+        return False
+    
+    cordova_mobilespec = os.path.join(BUILD_ROOT, "cordova-mobile-spec")
+    if not doCopy(os.path.join(BUILD_PARAMETERS.pkgpacktools, "cordova-mobile-spec"), cordova_mobilespec):
+        return False
+
+    orig_dir = os.getcwd()
+
+    os.chdir(BUILD_ROOT)
+    getsource_cmd = "git clone https://github.com/apache/cordova-coho.git"
+    if not doCMD(getsource_cmd, DEFAULT_CMD_TIMEOUT):
+        os.chdir(orig_dir)
+        return False
+
+    os.chdir(cordova_mobilespec)
+    output = commands.getoutput("git pull").strip("\r\n")
+    if output == "Already up-to-date.":
+        if not doCopy(os.path.join(BUILD_PARAMETERS.pkgpacktools, "mobilespec"), project_root):
+            return False
+    else:
+        os.chdir(os.path.join(cordova_mobilespec, "createmobilespec"))
+        install_cmd = "sudo npm install"
+        if not doCMD(install_cmd, DEFAULT_CMD_TIMEOUT):
+            os.chdir(orig_dir)
+            return False
+        os.chdir(BUILD_ROOT)
+        createmobilespec_cmd = "cordova-mobile-spec/createmobilespec/createmobilespec.js --android --global"
+        if not doCMD(createmobilespec_cmd, DEFAULT_CMD_TIMEOUT * 3):
+            os.chdir(orig_dir)
+            return False
+        os.chdir(project_root)
+        mv_cmd = "mv platforms/android/src/org/apache/mobilespec/MainActivity.java platforms/android/src/org/apache/mobilespec/mobilespec.java"
+        if not doCMD(mv_cmd, DEFAULT_CMD_TIMEOUT):
+            os.chdir(orig_dir)
+            return False
+        sed_cmd = "sed -i 's/MainActivity/mobilespec/g' `grep MainActivity -rl *`"
+        if not doCMD(sed_cmd, DEFAULT_CMD_TIMEOUT):
+            os.chdir(orig_dir)
+            return False
+    os.chdir(project_root)
+    add_webview_cmd = "cordova plugin add ../cordova-plugin-crosswalk-webview/"
```

Comment:
you should find the root of crosswalk webview plugin which is under tools/cordova_plugins/